examples/experimental/dagster-airlift/dagster_airlift/cli.py

```diff
@@ -0,0 +1,30 @@
+import click
+
+
+@click.group()
+def cli():
+    """Dagster Airlift CLI. Commands for interacting with the Dagster-Airlift package."""
+    pass
+
+
+@cli.group()
+def proxy() -> None:
+    """Commands for working with the Dagster-Airlift proxied state. Requires the `dagster-airlift[in-airflow]` package."""
+    try:
+        import dagster_airlift.in_airflow  # noqa
+    except:
+        raise Exception(
+            "dagster-airlift[in-airflow] must be installed in the environment to use any `dagster-airlift proxy` commands."
+        )
+
+
+@proxy.command()
+def scaffold():
+    """Scaffolds a proxied state folder for the current Airflow installation. Goes in the airflow dags folder."""
```

Comment:
might be worth noting that it marks all pipelines as not proxied, e.g. "Scaffolds a proxied state folder for the current Airflow installation, marking all tasks as not yet proxied."

Reply:
also mega nit but caps on Airflow & DAGs